### PR TITLE
Allow multiple --services-dir options

### PR DESCRIPTION
**Pull request by asherikov**

### CODE REVIEW

File: src/options-processing.cc [hunk 1]
```diff
@@ -27,32 +27,35 @@ const char * service_dir_opt::get_user_home()
...
-            std::memcpy(service_dir_w + user_home_len, "/dinit.d", dinit_d_len);
-            service_dir_w[full_len - 1] = 0;
-
-            service_dir = service_dir_w;
-            service_dir_dynamic = true;
+    if (service_dirs.empty()) {
+        bool home_service_dir_set = false;
+
+        /* service directory name */
+        if (not am_system_init) {
```
Comment (davmac314) [inline src/options-processing.cc:34]: For consistency prefer `!` instead of `not`
```suggestion
        if (!am_system_init) {
```

**Comment by davmac314** [comment]
The manual pages for dinit and dinitcheck also need to be updated, thanks!

**Comment by asherikov** [comment]
updated: style + man pages

**Comment by davmac314** [comment]
Merged. Thanks! If you want to be listed as a contributor, please open another pull request which updates the CONTRIBUTORS file.

**Pull request merged**